scripts/buttons.js

```diff
@@ -0,0 +1,107 @@
+import { getTransientState, getParks, getBizarreries, getEateries } from "./dataAccess.js"
+
+
+export const Buttons = () => {
+    const bizarreries = getBizarreries(),
+        parks = getParks(),
```

Comment:
Are these variables being declared?

Reply:
Yes, its shorthand. You can also it like this:
`const [one, two, three] = [1, 2, 3]`